saleor/account/__init__.py

```diff
@@ -0,0 +1,93 @@
+from django.utils.translation import pgettext_lazy
+
+
+class CustomerEvents:
+    """The different customer event types."""
+
+    # Account related events
+    ACCOUNT_CREATED = "account_created"
+    PASSWORD_RESET_LINK_SENT = "password_reset_link_sent"
+    PASSWORD_RESET = "password_reset"
+
+    # Order related events
+    PLACED_ORDER = "placed_order"  # created an order
+    NOTE_ADDED_TO_ORDER = "note_added_to_order"  # added a note to one of their orders
+    DIGITAL_LINK_DOWNLOADED = "digital_link_downloaded"  # downloaded a digital good
+
+    # Staff actions over customers events
+    CUSTOMER_DELETED = "customer_deleted"  # staff user deleted a customer
+    EMAIL_ASSIGNED = "email_assigned"  # the staff user assigned a email to the customer
+    NAME_ASSIGNED = "name_assigned"  # the staff user added set a name to the customer
+    NOTE_ADDED = "note_added"  # the staff user added a note to the customer
+
+    CHOICES = [
+        (
+            ACCOUNT_CREATED,
+            pgettext_lazy(
+                "Event from a customer that got their account created",
+                "The account account was created",
+            ),
+        ),
+        (
+            PASSWORD_RESET_LINK_SENT,
+            pgettext_lazy(
+                "Event from a customer that got the password reset link sent by email",
+                "The password email reset link was sent",
```

Comment:
I guess on the designs it is more like:

```suggestion
                "Password reset link was sent to the customer",
```